modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -6,129 +6,136 @@
       } else {
         logger.warn("CHECK THIS: ", trait, " data and prior are inconsistent:")
       }
-    } else if (p.data > 0.9995 | p.data < 0.0005) {
-      logger.debug("NOT OK! ", trait," data and prior are probably not the same:")
+    } else if (p.data > 0.9995 | p.data < 5e-04) {
+      logger.debug("NOT OK! ", trait, " data and prior are probably not the same:")
       return(NA)
     }
     logger.info(trait, "P[X<x] =", p.data)
   }
   
   ## Average trait data
-  trait.average <- sapply(jagged.data,
-                          function(x){mean(x$Y, na.rm = TRUE)})
+  trait.average <- sapply(jagged.data, function(x) mean(x$Y, na.rm = TRUE) )
   
   ## Set gamma distribution prior
-  prior.variances = as.data.frame(rep(1, nrow(prior.distns)))
-  row.names(prior.variances) = row.names(prior.distns)
-  prior.variances[names(trait.average), ] = 0.001 * trait.average^2 
-  prior.variances["seedling_mortality", 1] = 1.0
-  taupriors <- list(tauA = 0.01,
-                    tauB = apply(prior.variances, 1, function(x) min(0.01, x)))
+  prior.variances <- as.data.frame(rep(1, nrow(prior.distns)))
+  row.names(prior.variances) <- row.names(prior.distns)
+  prior.variances[names(trait.average), ] <- 0.001 * trait.average ^ 2
+  prior.variances["seedling_mortality", 1] <- 1
+  taupriors <- list(tauA = 0.01, tauB = apply(prior.variances, 1, function(x) min(0.01, x)))
   
   ### Run the meta-analysis
-  trait.mcmc  <- pecan.ma(jagged.data, prior.distns, taupriors, j.iter = iterations, 
-                          outdir = pft$outdir, random = random)
+  trait.mcmc <- pecan.ma(jagged.data,
+                         prior.distns,
+                         taupriors, 
+                         j.iter = iterations, 
+                         outdir = pft$outdir, 
+                         random = random)
+  
   ### Check that meta-analysis posteriors are consistent with priors
-  for(trait in names(trait.mcmc)){
-    post.median    <- median(as.matrix(trait.mcmc[[trait]][,'beta.o']))
-    prior          <- prior.distns[trait, ]
-    p.ma.post      <- p.point.in.prior(point = post.median, prior = prior)
+  for (trait in names(trait.mcmc)) {
+    post.median <- median(as.matrix(trait.mcmc[[trait]][, "beta.o"]))
+    prior       <- prior.distns[trait, ]
+    p.ma.post   <- p.point.in.prior(point = post.median, prior = prior)
+    
     ## if inside 95%CI, ok.
-    if(p.ma.post <= 0.9995 & p.ma.post >= 0.0005){
+    if (p.ma.post <= 0.9995 & p.ma.post >= 5e-04) {
       if (p.ma.post <= 0.975 & p.ma.post >= 0.025) {
         logger.info("OK! ", trait, " posterior and prior are consistent:")
       } else {
         logger.warn("CHECK THIS: ", trait, " posterior and prior are inconsistent:")
       }
-    } else if (p.ma.post > 0.9995 | p.ma.post < 0.0005) {
-      logger.severe("NOT OK! ", trait," posterior and prior are probably not the same:")
+    } else if (p.ma.post > 0.9995 | p.ma.post < 5e-04) {
```

Comment:
More generally, this block is almost identical to the once above. It would be better to wrap it in its own function and call it twice. DRY.